packages/epo-widget-lib/src/atomic/Blinker/Images/Images.tsx

```diff
@@ -22,11 +24,15 @@ const Images: FunctionComponent<ImagesProps> = ({
 
   useEffect(() => {
     setIsLoading(imagesLoaded !== images.length);
-  }, [imagesLoaded]);
+
+    if (!isLoading) {
+      loadedCallback && loadedCallback();
+    }
+  }, [imagesLoaded, isLoading]);
 
   return (
     <Styled.BlinkContainer data-testid="blinker-images" className={className}>
-      <CircularLoader isVisible={true} />
+      {!imagesLoaded && <CircularLoader isVisible={isLoading} />}
```

Comment:
setting this to remove from the DOM when images have loaded so there is not an aria live region showing "loading" perpetually